x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -627,6 +644,7 @@ export async function ensurePackagesCompletedInstall(
             savedObjectsClient,
             pkgkey,
             esClient,
+            spaceId: (pkg.attributes.installed_kibana_space_id as string) || DEFAULT_SPACE_ID,
```

Comment:
nit: why is it needed to add `as string`?

Reply:
Good spot, the casting wasn't needed not sure why I had it there 🤷